tox.ini

```diff
@@ -106,11 +111,11 @@ commands =
     checkformatting: black --check h tests bin
     checkformatting: isort --quiet --check-only h tests bin
     {tests,functests}: sh bin/create-testdb
-    tests: coverage run -m pytest {posargs:tests/h/}
+    tests: python -m pytest --cov --cov-report= --cov-fail-under=0 --numprocesses logical --dist loadgroup {posargs:tests/h/}
```

Comment:
Same `pytest` args as LMS uses:

* `--cov`: needed to enable `pytest-coverage`, even with the other `--cov-*` options being here
* `--cov-report=` (with no argument) disables coverage reporting. We don't want `make test` (or `tox -e tests` etc) to print out a coverage report. Especially not when you only run certain tests (`tox -e tests tests/unit/foo_test.py`), it's really annoying. Instead, `make coverage`, `make sure`, or `tox -e coverage` will print the coverage report
* `--cov-fail-under`: we also don't want `make test` (or `tox -e tests` etc) to fail if coverage is not 100%, especially not when you only run certain tests (`tox -e tests tests/unit/foo_test.py`). Instead, `make coverage`, `make sure`, or `tox -e coverage` will still fail if coverage is not 100%.

The one thing that's new here is `--dist-loadgroup` which enables the `pytest-xdist` grouping feature that I'm using to avoid running Elasticsearch tests in parallel with each other.